packages/nifti-volume-loader/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@cornerstonejs/nifti-volume-loader",
+  "version": "1.4.1",
+  "description": "",
+  "main": "dist/umd/index.js",
+  "types": "dist/esm/index.d.ts",
+  "module": "dist/esm/index.js",
+  "repository": "https://github.com/cornerstonejs/cornerstone3D",
+  "files": [
+    "dist/"
+  ],
+  "directories": {
+    "test": "test"
+  },
+  "sideEffects": false,
+  "scripts": {
+    "build:cjs": "tsc --project ./tsconfig.cjs.json",
```

Comment:
I thought we were just doing two of the builds now?  I wouldn't do all three if we are shortly dropping one.